app/components/AmountsPanel/index.jsx

```diff
@@ -1,32 +1,89 @@
 // @flow
-
 import React from 'react'
+import classNames from 'classnames'
+import { orderBy, times } from 'lodash-es'
 
 import AmountsInfoBox from './AmountsInfoBox'
 import { CURRENCIES } from '../../core/constants'
 
+import Nothing from '../../assets/icons/nothing.svg'
+
+import amountBoxStyles from './AmountsInfoBox/AmountsInfoBox.scss'
 import styles from './AmountsPanel.scss'
 
+type AmountDataItem = {
+  symbol: string,
+  price: string,
+  totalBalance: string,
+  totalBalanceWorth: number
+}
+
 type Props = {
-  amountsData: Array<*>,
+  amountsData: AmountDataItem[],
   currencyCode: string
 }
 
-const AmountsPanel = ({ amountsData, currencyCode }: Props) => (
-  <section className={styles.amountsPanel}>
-    {amountsData.map(dataset => (
-      <AmountsInfoBox
-        key={dataset.symbol}
-        assetName={dataset.symbol}
-        assetPrice={dataset.price}
-        totalAmount={dataset.totalBalance}
-        remainingAmount={dataset.currentBalance}
-        totalBalanceWorth={dataset.totalBalanceWorth}
-        remainingBalanceWorth={dataset.remainingBalanceWorth}
-        fiatCurrencySymbol={CURRENCIES[currencyCode].symbol}
-      />
-    ))}
-  </section>
-)
+const ORDER_BY_FIELD = 'totalBalanceWorth'
+const ORDERY_BY_DIRECTION = 'asc'
```

Comment:
dont we want these in descending order of total balance worth not ascending? User probably expects to see their most valuable asset first